plugins/linux-pipewire/pipewire.c

```diff
@@ -311,6 +312,8 @@ static inline struct spa_pod *build_format(obs_pipewire_stream *obs_pw_stream,
 	struct spa_fraction min_framerate;
 	struct spa_fraction framerate;
 
+	spa_pod_dynamic_builder_init(&pod_builder, NULL, 0, 4096);
```

Comment:
spa_pod_dynamic_builder_clean is referenced in other places. I cant find any docs on spa_pod_dynamic_builder (not sure why i hoped it would exist), but can you confirm that we dont need to clean up after init.